### PR TITLE
Move CI to GH Actions

### PR DESCRIPTION
**Pull request by willis102**
Move CI for preview / deploy of developer docs site to use Github Actions instead of Travis.

This will need to be updated to use the appropriate commit on tk-doc-generator once the corresponding change to that repo is merged.

### CODE REVIEW

**Comment by toolkitty** [comment]
[Documentation Preview](http://sg-devdocs.s3-website-us-east-1.amazonaws.com/tk-doc-generator/360e4df28cfaacd500bc5c4d3d1c2f83fd46765c/index.html)

**Comment by toolkitty** [comment]
[Documentation Preview](http://sg-devdocs.s3-website-us-east-1.amazonaws.com/tk-doc-generator/b0377792a8b8e3da6aabe595a332338299469065/index.html)

**Comment by toolkitty** [comment]
[Documentation Preview](http://sg-devdocs.s3-website-us-east-1.amazonaws.com/tk-doc-generator/b0377792a8b8e3da6aabe595a332338299469065/index.html)

**Comment by toolkitty** [comment]
[Documentation Preview](http://sg-devdocs.s3-website-us-east-1.amazonaws.com/tk-doc-generator/905a2f17caa92cdd730462bcf6bb47bad0c8405d/index.html)

File: .github/workflows/generate-preview.yml [hunk 1]
```diff
@@ -0,0 +1,35 @@
+name: Generate Preview
```
Comment (jfboismenu) [inline .github/workflows/generate-preview.yml:1]: Could you add the copyright notice please?

File: .github/workflows/publish-docs.yml [hunk 2]
```diff
@@ -0,0 +1,48 @@
+name: Publish Developer Docs
```
Comment (jfboismenu) [inline .github/workflows/publish-docs.yml:1]: Could you add the copyright notice please?

**Review by jfboismenu — APPROVED** [review]
Looks good to me. Nice job.

**Comment by toolkitty** [comment]
[Documentation Preview](http://sg-devdocs.s3-website-us-east-1.amazonaws.com/tk-doc-generator/1756598e98e68eddf80ac5de5a296f039685b499/index.html)

**Comment by toolkitty** [comment]
[Documentation Preview](http://sg-devdocs.s3-website-us-east-1.amazonaws.com/tk-doc-generator/1c1113164aece7c72b8906555c9ec42b47f5cca8/index.html)

**Pull request merged**